lib/AST/ASTPrinter.cpp

```diff
@@ -2357,7 +2357,21 @@ void PrintAST::visitTopLevelCodeDecl(TopLevelCodeDecl *decl) {
 }
 
 void PrintAST::visitIfConfigDecl(IfConfigDecl *ICD) {
-  // FIXME: Pretty print #if decls
+  if (!Options.PrintIfConfig)
+    return;
+
+  for (auto &Clause : ICD->getClauses()) {
+    if (&Clause == &*ICD->getClauses().begin())
+      Printer << tok::pound_if << " "; // FIXME: print condition
```

Comment:
Can you print *something* here, even a comment, so that people will know it's not a bug? Or, well, that it's a *known* bug?